.github/workflows/windows.yml

```diff
@@ -0,0 +1,21 @@
+name: Windows build
+
+on: push, pull_request
+
+env:
+  # Customize the env if
+  BUILD_TYPE: Release
+  DPDK_VERSION: 22.07
+
+jobs:
+  Build:
+    #if: github.repository == 'OpenVisualCloud/Media-Transport-Library'
+    runs-on: [Windows, self-hosted]
+
+    steps:
+    - name: checkout kahawai repo
+      uses: actions/checkout@v3
+    
+    - name: Build
+      run: C:\actions-runner\windows_installer.ps1
```

Comment:
Is it possible extract the steps from this script? It's better if we can manage the build steps by yml file.